rialto_airflow/dags/harvest.py

```diff
@@ -123,9 +124,18 @@ def distill_publications(snapshot, openalex_fill_in, dimensions_fill_in):
         """
         Distill the publication metadata into publication table columns.
         """
-        distill.distill(snapshot)
+        count = distill.distill(snapshot)
 
-        return snapshot
+        return count
```

Comment:
This is the new task that will iteratively look at all publications with dimensions data to link them to the funder table.